src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Creates a class of TaskList
+        TaskList taskList = new TaskList(100);
+        String line;
+        Scanner in = new Scanner(System.in);
+        boolean userSaidBye = false;
+        // Welcomes user
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Hello! I'm PoratoBot");
+        System.out.println("    How can I assist you mortal?");
+        System.out.println("    ____________________________________________________________");
+        while (!userSaidBye) {
+            // Takes in user input
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                userSaidBye = true;
+            } else if(line.equals("list")) {
+                // Prints the all the tasks in the list
+                System.out.println("    ____________________________________________________________");
+                System.out.println("     Here are the tasks in your list:");
+                taskList.listTasks();
+                System.out.println("    ____________________________________________________________");
+            } else if (line.startsWith("mark")){
+                // Marks the task in the list
+                String[] sentence = line.split(" ");
+                System.out.println("    ____________________________________________________________");
+                System.out.println("     Nice! I've marked this task as done:");
+                taskList.checkTask(Integer.parseInt(sentence[1]));
+                System.out.println(" ");
+                System.out.println("    ____________________________________________________________");
+            } else if (line.startsWith("unmark")){
+                // Unmarks the task in the list
+                String[] sentence = line.split(" ");
+                System.out.println("     OK, I've marked this task as not done yet:");
+                taskList.uncheckTask(Integer.parseInt(sentence[1]));
+                System.out.println("    ____________________________________________________________");
+            } else if(line.startsWith("deadline")){
+                // Adds a deadline task into the list
+                String []sentence = line.split("deadline|/by");
+                Task newTask = new Deadlines(sentence[1], sentence[2]);
+                taskList.addTask(newTask);
+                System.out.println("    ____________________________________________________________");
+                System.out.println("     Got it. I've added this task:");
+                System.out.println("       " + newTask);
+                System.out.println("     Now you have " + taskList.getNoOfTasks() + " tasks in the list.");
+                System.out.println("    ____________________________________________________________");
```

Comment:
These lines to print are repeated throughout the conditional structure, perhaps you can use inheritance to perform this?